src/Tribe/Attendees.php

```diff
@@ -935,4 +938,45 @@ public function update_attendee( $attendee, $attendee_data ) {
 		return $provider->update_attendee( $attendee, $attendee_data );
 	}
 
+	/**
+	 * Generate the export URL for exporting attendees
+	 *
+	 * @since TBD
+	 *
+	 *
+	 * @return string Relative URL for the export.
+	 */
+	public function get_export_url (){
+		return  add_query_arg(
+			[
+				'attendees_csv'       => true,
+				'attendees_csv_nonce' => wp_create_nonce( 'attendees_csv_nonce' ),
+			]
+		);
+	}
+	/**
+	 * Echo the button for the export that appears next to the attendees page title.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $event_id
+	 * @param Tribe__Tickets__Attendees $attendees
+	 *
+	 * @return string Relative URL for the export.
+	 */
+	public function include_export_button_title( $event_id, Tribe__Tickets__Attendees $attendees ){
+
+		// Bail early if there are no attendees
+		if ( ! $attendees->attendees_table->has_items() ) {
+			return;
+		}
+
+		// Bail early if user is not owner/have permissions
+		if ( ! $this->user_can_manage_attendees( 0, $event_id ) ) {
+			return;
+		}
+
+		echo sprintf( '<a target="_blank" href="%s" class="export action page-title-action">%s</a>', esc_url( $export_url = $this->get_export_url() ), esc_html__( 'Export', 'event-tickets' ) ) ;
```

Comment:
For security issues we must have `rel="noopener noreferrer"` when the `target` is `_blank`

Reply:
We could possibly split this into multiple lines for better readability.